### PR TITLE
Check object exists before pushing History Tab to BaseElement

### PR DESCRIPTION
**Pull request by eehondas**
Hello,
This is a quick fix relative to issue #390 .
I tested it and it does prevent the history tab from showing (duh!).
I used a separate logic block rather than adding an && condition because I assume the inner logic block will be removed in future instances and this way you can just take away the inner condition. Not rocket science obviously.

Thanks for the module again.

### CODE REVIEW

File: src/Models/BaseElement.php [hunk 1]
```diff
@@ -300,15 +300,18 @@ public function getCMSFields()
...
-            if (class_exists(HistoryViewerField::class)) {
-                Requirements::javascript('dnadesign/silverstripe-elemental:client/dist/js/bundle.js');
-
-                $historyViewer = HistoryViewerField::create('ElementHistory');
-                $fields->addFieldToTab('Root.History', $historyViewer);
-
-                $fields->fieldByName('Root.History')
-                    ->addExtraClass('elemental-block__history-tab tab--history-viewer');
+            // Check object exists before pushing History Tab
+            if($this->ID) {
```
Comment (ScopeyNZ) [inline src/Models/BaseElement.php]: Can you use `$this->isInDB()` here? It's pretty similar logic but the method might be updated to more accurately determine if the object is persisted.

Comment (ScopeyNZ) [reply src/Models/BaseElement.php]: Also, combining the conditions is nicer because then you're only updated one line instead of 12.

**Comment by ScopeyNZ** [comment]
Thanks for the PR @eehondas! I left a few requests and the automated tests are only failing due to some PHPCS (linting) failures pasted here for clarity:

```
 304 | ERROR | [x] Expected 1 space(s) after IF keyword; 0 found
 306 | ERROR | [x] Line indented incorrectly; expected 16 spaces, found 14
 307 | ERROR | [x] Line indented incorrectly; expected at least 20 spaces
     |       |     found 18
 309 | ERROR | [x] Line indented incorrectly; expected at least 20 spaces
     |       |     found 18
 310 | ERROR | [x] Line indented incorrectly; expected at least 20 spaces
     |       |     found 18
 312 | ERROR | [x] Line indented incorrectly; expected at least 20 spaces
     |       |     found 18
 314 | ERROR | [x] Line indented incorrectly; expected 16 spaces, found 14
```

If you wouldn't mind fixing that up I can get this merged for you 🙂

**Comment by eehondas** [comment]
Sure will, sorry, I thought the different lines would make life easier but it's actually the opposite.

**Pull request merged**